articles/backup/backup-create-rs-vault.md

```diff
@@ -17,7 +17,7 @@ Azure Backup automatically handles storage for the vault. You need to specify ho
 > [!NOTE]
 > Changing **Storage Replication type** (Locally redundant/ Geo-redundant) for a Recovery Services vault has to be done before configuring backups in the vault. Once you configure backup, the option to modify is disabled.
 >
->- If you haven't yet configured the backup, then [follow these steps](#set-storage-redundancy) to review and modify the settings.
+>- If you haven't yet configured the backup, then follow the steps below.
```

Comment:
If you haven't yet configured the backup, follow the steps below to review and modify the settings.